src/views/Affiliates/cards/AffiliateProgressCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { AFFILIATES_REQUIRED_VOLUME_USD } from '@/constants/affiliates';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { Output, OutputType } from '@/components/Output';
+
+export const AffiliateProgressCard = ({
+  volume = 0,
+  className,
+}: {
+  volume?: number;
+  className?: string;
+}) => {
+  const stringGetter = useStringGetter();
+
+  const progressPercent = ((volume / AFFILIATES_REQUIRED_VOLUME_USD) * 100).toFixed(0).toString();
+  const remaining = AFFILIATES_REQUIRED_VOLUME_USD - volume;
+  return (
+    <div className={className} tw="flex flex-col gap-1.5 rounded-1 p-1">
+      <div tw="flex items-center justify-between gap-0.5">
+        <div tw="flex flex-col gap-0.375">
+          <div tw="font-semibold">{stringGetter({ key: STRING_KEYS.BECOME_AN_AFFILIATE })}</div>
+          <div tw="text-color-text-0">
+            {stringGetter({ key: STRING_KEYS.TRADE_MORE_VOLUME_REQUIREMENT })}
+          </div>
+        </div>
+        <div tw="text-large font-medium">{progressPercent}%</div>
```

Comment:
`<Output tw="inline" type={OutputType.Percent} value={progressPercent} fractionDigits={0} />` and can remove the `* 100` in `progressPercent`.

Reply:
positives would be no need to `.toString()` or `.toFixed()`

Reply:
omg isnt this so much more verbose 💀

Reply:
more verbose but keeps it consistent w/ other number outputs. i.e. mono font-feature-settings, not needing to add `%` suffix, rounding mode, etc.

Reply:
omg im so against this but ill do it